08_book_titles/book.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Book
+  attr_reader :title
+
+  def title=(new_title)
+    words = new_title.split(' ')
+    words = [words[0].capitalize] +
```

Comment:
first of all `first` instead of `[0]`
secondly, everything after `+` to separate method